c-cannons/bangazon-api
# Serialize

### Pull Requests

#### Descriptions
 - added serializer to Gemfile
- built Order Serializer and Product Serializer
- Reset Order associations

#### Steps to Test
- to add serializer gem, run this in terminal `bundle install`
- make sure to seed your database correctly, chuds.  in order.

#### Link to Feature Ticket
https://github.com/c-cannons/bangazon-api/issues/10
